portfolio/src/main/java/com/google/sps/servlets/DeleteServlet.java

```diff
@@ -35,12 +35,14 @@
 
   @Override
   public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    Query query = new Query("Comment").addSort("time", SortDirection.DESCENDING);
+    long messageId = Long.parseLong(request.getParameter("messageId"));
```

Comment:
String constants like "messageId" would preferably be private/public static final class member variables.